docs/kubernetes/flannel-bigip-info.rst

```diff
@@ -1,143 +1,116 @@
+
+The BIG-IP FDB record created for this Node would look like this ::
+
+   flannel_vxlan {
+    records [
+       98:ba:76:dc:54:fe {
+         endpoint: 172.16.2.10
        }
+    ]
+   }
 
-These configurations performed by the F5 BIG-IP Controller informs the BIG-IP device about where
-and how to forward traffic to the node "10.244.1.2".
- 
-The next step is to define a tunnel self-ip on the BIG-IP device that will live in a Flannel subnet.
-At this point there should be Flannel subnets (podCIDR) defined for each Node in Kubernetes.
-We need to decide on a similar subnet that will not collide with any other Node's subnet.
-We'll say this subnet is 10.244.30.0/24. Our BIG-IP device's self-ip will live within this subnet;
-we will create one for our VXLAN tunnel at 10.244.30.15/16. The subnet mask /16 matches
-the default subnet mask of the Flannel network range.
- 
-At this point, our BIG-IP device knows how to route to the Kubernetes network,
-but is not yet a part of it. That's Flannel's job. We need to tell Flannel about the BIG-IP device.
- 
-Earlier we mentioned that Flannel knows about the Kubernetes Nodes via the Kubernetes API.
-This means that in order for Flannel to add the BIG-IP device to the Kubernetes network,
-the Kubernetse API must know about the BIG-IP device. How can we possibly do that?
-By adding the BIG-IP device as a Node in Kubernetes, of course!
-This may seem complicated, but is actually quite simple. Flannel simply needs to see the
-BIG-IP device's information in the supported Flannel annotations, and a defined subnet (podCIDR)
-on the Node, and "voila!", the BIG-IP device will be able to participate in the VXLAN.
+and the static ARP entry would look like this: ::
 
-Here is what the BIG-IP device Node looks like:
+   {
+      name: k8s-10.244.1.2
+      ipaddress: 10.244.1.2
+      macaddress: 98:ba:76:dc:54:fe
+   }
 
-  .. literalinclude:: /kubernetes/config_examples/f5-kctlr-bigip-node.yaml
-     :linenos:
+Together, these records tell the BIG-IP device what Kubernetes resource(s) should receive traffic from the node whose IP address is "10.244.1.2".
 
-Flannel sees the required annotations on this Node, and adds the BIG-IP device to the VXLAN.
- 
-With all of these pieces in place, traffic sent from (or through) the BIG-IP device,
-directed at the Kubernetes Pod, will successfully reach its destination!
+Tell flannel about the BIG-IP
+`````````````````````````````
+
+At this point, your BIG-IP device knows how to route to the Kubernetes network, but it isn't a part of the network yet. You'll need to set up flannel to be aware of the BIG-IP device. To do so, you'll create an additional flannel subnet and allocate a self IP address from this subnet as the BIG-IP's VTEP.
+
+When setting up a new subnet to use for the BIG-IP, make sure that it doesn't collide with any of the Node subnets.
+
+**For example:**
+Create a new subnet: ``10.244.30.0/24``. Your BIG-IP device's self IP will live within this subnet.
```

Comment:
I think the word "define" makes more sense than "create" here. The user isn't actually creating anything, but they will have to "come up with" a subnet range that gets put into the Node configuration (podCIDR) at a later step.

Reply:
Gotcha. I was thinking it was like the Host Subnet in OpenShift.